apps/emqx/src/emqx_misc.erl

```diff
@@ -545,10 +545,23 @@ readable_error_msg(Error) ->
                 {ok, Msg} ->
                     Msg;
                 false ->
-                    iolist_to_binary(io_lib:format("~0p", [Error]))
+                    to_hr_error(Error)
             end
     end.
 
+to_hr_error(nxdomain) ->
+    <<"Host not found">>;
```

Comment:
```suggestion
    <<"Host name not resolvable">>;
```

Reply:
I used "Host not found" since it's what unix system's error message look like for that case.